classes/ObjectModel.php

```diff
@@ -999,7 +999,7 @@ public function validateFieldsLang($die = true, $errorReturn = false)
 
             // If the object has not been loaded in multilanguage, then the value is the one for the current language of the object
             if (!is_array($values)) {
-                $values = [$this->id_lang => $values];
+                $values = [$this->id_lang ?? '' => $values];
```

Comment:
```suggestion
                $values = [$this->id_lang ?? 0 => $values];
```
No? since it_lang is supposed to be an integer, this seems like a more fitting fallback But even this one is not really ok, if there is no language id the validation cannot work properly anyway

Reply:
Fixed :)